pos_search_accented_unaccented/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRLfs
+#   - Vincent Van Rossem <vincent@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "POS Search Accented/Unaccented Characters",
+    "version": "9.0.1.0.0",
+    "depends": [
+        "point_of_sale",
+    ],
```

Comment:
do this module have a prerequisite on the postgres package installation ?

https://www.odoo.com/fr_FR/forum/aide-1/question/how-to-search-without-accented-characters-1229

Reply:
I guess no prerequisite since I haven't installed nor launched `unaccent` on my localhost server, whereas it is installed/launched on our testing server.